backend/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.9
+RUN mkdir -p /app
+WORKDIR /app
+COPY . .
+
+RUN pip install --upgrade pip
+RUN pip install -e .[dev]
+RUN pip install fastapi[all]
+RUN pip install -i https://test.pypi.org/simple/ flotilla-openapi
+
+EXPOSE 8000
+RUN pip install gunicorn
```

Comment:
```suggestion
```
This may be removed when using `uvicorn` and `uvicorn` is installed in the dependencies already.